receiver/hostmetricsreceiver/internal/scraper/diskscraper/disk_metadata.go

```diff
@@ -66,6 +66,20 @@ var diskTimeDescriptor = func() pdata.Metric {
 	metric := pdata.NewMetric()
 	metric.InitEmpty()
 	metric.SetName("system.disk.time")
+	metric.SetDescription("Time disk spent activated.")
```

Comment:
Should we call out the fact that this is the total (both read and write) time?

Reply:
I think that's probably implied by the fact there is no direction label? Do you think that's not clear enough?